galaxy/Dockerfile

```diff
@@ -86,91 +92,129 @@ RUN groupadd -r postgres -g $GALAXY_POSTGRES_GID \
+    && chown -R $GALAXY_USER:$GALAXY_USER $GALAXY_ROOT_DIR \
     # Setup Galaxy configuration files.
     && mkdir -p $GALAXY_CONFIG_DIR $GALAXY_CONFIG_DIR/web \
     && chown -R $GALAXY_USER:$GALAXY_USER $GALAXY_CONFIG_DIR \
-    && rm -rf /tmp/* /root/.cache/ /var/cache/* $GALAXY_ROOT/client/node_modules/ $GALAXY_VIRTUAL_ENV/src/ /home/galaxy/.cache/ /home/galaxy/.npm \
-    && su $GALAXY_USER -c "cp $GALAXY_ROOT/config/galaxy.yml.sample $GALAXY_CONFIG_FILE" \
+    && rm -rf /tmp/* /root/.cache/ /var/cache/* $GALAXY_ROOT_DIR/client/node_modules/ $GALAXY_VIRTUAL_ENV/src/ /home/galaxy/.cache/ /home/galaxy/.npm \
+    && su $GALAXY_USER -c "cp $GALAXY_ROOT_DIR/config/galaxy.yml.sample $GALAXY_CONFIG_FILE" \
+    && su $GALAXY_USER -c "cp $GALAXY_ROOT_DIR/config/tool_conf.xml.sample $GALAXY_CONFIG_TOOL_CONFIG_FILE" \
     # cleanup dance
-    && find $GALAXY_ROOT -name '*.pyc' -delete | true \
+    && find $GALAXY_ROOT_DIR -name '*.pyc' -delete | true \
     && find /usr/lib/ -name '*.pyc' -delete | true \
     && find $GALAXY_VIRTUAL_ENV -name '*.pyc' -delete | true \
-    && rm -rf /tmp/* /root/.cache/ /var/cache/* $GALAXY_ROOT/client/node_modules/ $GALAXY_VIRTUAL_ENV/src/ /home/galaxy/.cache/ /home/galaxy/.npm
+    && rm -rf /tmp/* /root/.cache/ /var/cache/* $GALAXY_ROOT_DIR/client/node_modules/ $GALAXY_VIRTUAL_ENV/src/ /home/galaxy/.cache/ /home/galaxy/.npm
 
 
-ADD ./reports_wsgi.ini.sample $GALAXY_CONFIG_DIR/reports_wsgi.ini
+ADD ./reports.yml.sample $GALAXY_CONFIG_DIR/reports.yml
 ADD sample_tool_list.yaml $GALAXY_HOME/ephemeris/sample_tool_list.yaml
-ADD roles/ /ansible/roles
-ADD provision.yml /ansible/provision.yml
-ADD postgresql_provision.yml /ansible/postgresql_provision.yml
 
-## This is playbook is modifying/installing python2 and python3 stuff
-RUN apt update -qq && apt install --no-install-recommends -y ansible dirmngr gpg gpg-agent \
-    && ansible-playbook /ansible/postgresql_provision.yml \
-    && apt purge ansible dirmngr gpg gpg-agent -y && apt-get autoremove -y && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-    # cleanup dance
-    && find $GALAXY_ROOT/ -name '*.pyc' -delete | true \
+# Activate Interactive Tools during runtime
+ADD ./tools_conf_interactive.xml.sample $GALAXY_INTERACTIVE_TOOLS_CONFIG_FILE
+
+RUN chown $GALAXY_USER:$GALAXY_USER $GALAXY_CONFIG_DIR/reports.yml $GALAXY_HOME/ephemeris/sample_tool_list.yaml $GALAXY_INTERACTIVE_TOOLS_CONFIG_FILE
+RUN chmod 0644 $GALAXY_CONFIG_DIR/reports.yml $GALAXY_HOME/ephemeris/sample_tool_list.yaml $GALAXY_INTERACTIVE_TOOLS_CONFIG_FILE
+
+# Make python3 standard
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+
+ADD roles/ /ansible/roles
+ADD ansible/ /ansible/
+
+# Install ansible and related dependencies
+RUN apt update -qq && apt install --no-install-recommends -y software-properties-common dirmngr gpg gpg-agent \
+    && sudo add-apt-repository ppa:ansible/ansible \
```

Comment:
Not sure which version of ansible you are installing here, but I get:

```
190.5 ERROR! [DEPRECATED]: ansible.builtin.include has been removed. Use include_tasks or import_tasks instead. This feature was removed from ansible-core in a release after 2023-05-16. Please update your playbooks.
190.5 
190.5 The error appears to be in '/ansible/roles/galaxyprojectdotorg.galaxyextras/tasks/main.yml': line 2, column 3, but may
190.5 be elsewhere in the file depending on the exact syntax problem.
190.5 
190.5 The offending line appears to be:
190.5 
190.5 
190.5 - include: galaxy_root.yml
190.5   ^ here
````

later below.